floodviz/tests/test_hydrograph_utils.py

```diff
@@ -194,6 +169,12 @@ def test_valid_data_missing_count(self):
     def test_valid_data_missing_value(self):
         ret = parse_hydrodata(self.mock_data_missing_points)
         for dp in ret:
-            if dp.get('name') is 'no_data':
-                self.assertEqual(dp.get('value'), 'NA')
+            try:
```

Comment:
You should know exactly where the dummy data will be in the list. You should look for it there. This test would succeed if the first element in the list was null for instance and doesn't check to see if the values where filled in every 15 minutes for the gap.